src/setup.py

```diff
@@ -26,6 +26,7 @@
         'scipy<1.3.0', # openfermion 0.10.0 is incompatible with scipy 1.3 because of comb function
         'openfermion>=0.10.0',
         'python-rapidjson',
+        'pyquil>=2.17.0',
```

Comment:
You don't need dependency on `pyquil` since it's already in `z-quantum-core`.
Actually I'm not sure if we shouldn't duplicate all the required dependencies, but perhaps without versioning? 🤔

Reply:
For this problem, I was imagining putting dependencies that are explicitly relied on in the packages themselves. Because if we, for instance, updated `z-quantum-core` to not need `pyquil`, then this would be a breaking change to this resource.